src/plugin-store.svelte

```diff
@@ -97,49 +99,47 @@ function toggleOfficialPlugin(plugin: Plugin, isEnabled: boolean) {
 let externalPlugins: Plugin[] = [];
 
 async function fetchExternalPlugins() {
-	const externalPluginsMockup: Plugin[] = [
-		{
-			name: "Type Designer",
-			author: "SprintEins",
-			src: "https://sprinteins.github.io/oscd-plugins/type-designer/index.js",
-			kind: "editor",
-			installed: false,
-		},
-		{
-			name: "Auto Doc",
-			author: "SprintEins",
-			src: "https://sprinteins.github.io/osdc-plugins/auto-doc/index.js",
-			kind: "editor",
-			installed: false,
-		},
-	];
+	const url = "https://sprinteins.github.io/oscd-plugin-store/plugins.json";
```

Comment:
We can use a relative url, like how we load the styles.
Otherwise we always have to change this based on deployment.